src/app/app-platform/ContentAppPlatform.cpp

```diff
@@ -755,6 +756,19 @@ CHIP_ERROR ContentAppPlatform::ManageClientAccess(Messaging::ExchangeManager & e
             {
                 // notify content app about this nodeId
                 app->AddClientNode(subjectNodeId);
+
+                auto tempAccountId = CharSpan::fromCharString(rotatingId.c_str());
+                auto setupPIN = CharSpan::fromCharString(std::to_string(passcode).c_str());
```

Comment:
This is use-after-free right here.

Reply:
The lifetimes should be valid here as far as I can tell, but either way this logic is removed in latest update.